internal/test/test.go

```diff
@@ -113,6 +113,38 @@ func testCode(
 	runner := cdcTests.NewTestRunner()
 	if coverageEnabled {
 		coverageReport = runtime.NewCoverageReport()
+		contracts := map[string]string{
```

Comment:
**NOTE:** I am sure there's a way to get these mapping dynamically, I have just hardcoded it here to illustrate my point. We'll update it to dynamically fetch all the available system contracts, as new ones we'll probably be introduced in the future.

Reply:
This might be better suitable for the `cadence-tools/test` repository.

Reply:
Update: Opened https://github.com/onflow/cadence-tools/pull/110 in favor of this.